blas64/blas64.go

```diff
@@ -178,20 +180,23 @@ func Rotmg(d1, d2, b1, b2 float64) (p blas.DrotmParams, rd1, rd2, rb1 float64) {
 	return blas64.Drotmg(d1, d2, b1, b2)
 }
 
-// Rot applies a plane transformation.
-//  x[i] = c * x[i] + s * y[i]
-//  y[i] = c * y[i] - s * x[i]
+// Rot applies a plane transformation to n points represented by the vectors x
+// and y:
+//  x[i] =  c * x[i] + s * y[i],
+//  y[i] = -s * x[i] + c * y[i], for all i.
 func Rot(n int, x, y Vector, c, s float64) {
 	blas64.Drot(n, x.Data, x.Inc, y.Data, y.Inc, c, s)
 }
 
-// Rotm applies the modified Givens rotation to the 2×n matrix.
+// Rotm applies the modified Givens rotation to n points represented by the
+// vectors x and y.
 func Rotm(n int, x, y Vector, p blas.DrotmParams) {
 	blas64.Drotm(n, x.Data, x.Inc, y.Data, y.Inc, p)
 }
 
-// Scal scales x by alpha.
-//  x[i] *= alpha
+// Scal computes
+//  x = alpha * x,
```

Comment:
Not sure. I had two motivations for this change. I like the mathematically oriented comments of the level 2 and 3 routines and I wanted to use it also in level 1. And the original comment says the same thing twice (this is probably true also for other level 1 functions).

I propose either
```
// Scal scales the vector x by alpha.
```
or
```
// Scal computes
//  x[i] *= alpha for all i.
```